launch

```diff
@@ -546,7 +586,11 @@ def main():
     if emscripten_path == '': return 1
 
   Log.info('Use emscripten at %s' % emscripten_path)
-  pathlist = [grunt_path, emscripten_path, os.path.join(tool_path, 'WebIDLBinder'), '']
+
+  if emscripten_path != '':
```

Comment:
IMO, this line is unnecessary as emscripten_path already checked on above.

Reply:
I agreed. Fixed.